gravitee-apim-portal-webui-next/src/components/footer/footer.component.html

```diff
@@ -16,4 +16,4 @@
 
 -->
 <app-company-title [title]="title" />
-<p>© 2024 {{ company }}. All rights reserved.</p>
+<p>© 2024. All rights reserved.</p>
```

Comment:
I know it's not the purpose of the PR, but could it be possible to replace 2024 with the current year?
Here is how it's done in the current portal:
https://github.com/gravitee-io/gravitee-api-management/blob/5a0e988e91e1274ff52823dd863e75628d6c3fa6/gravitee-apim-portal-webui/src/app/app.component.ts#L92
https://github.com/gravitee-io/gravitee-api-management/blob/683ac06eaffd352a4364ece0b7c56e53909f4c47/gravitee-apim-portal-webui/src/app/app.component.html#L104